CMSgov/ato-blueprint
# Feature/projects page

Story: https://jiraent.cms.gov/browse/ISPGBSS-730
Figma: https://www.figma.com/file/9a26GjOwssW76cTr01hfXL/Guidelines-%26-Wireframes?node-id=124%3A741

### Changes
- converted design on project page to match the Figma (except the "View Project Team" link since we don't yet have that as a functional feature)
- created a guidedmodules/utils file with a function to get the project's acronym
- created a function in controls/utils file to return an object containing control compliance stats (`controls_addressed_count`, `total_controls_count`, and `percent_compliant`)
- changed pagination of projects page to display 9 projects per page
- misc cleanup

### Testing
- Visit the /projects page and verify that it matches the Figma designs
- Verify that the project cards display information consistent with each project (name, acronym, percent compliant bar)
- Verify the functionality of the project cards (list of links, view project button)
- Regression check a project page for the percent compliant related information and the project acronym. Also regression check a project settings page for the project acronym. (since this functionality was moved into utils files)
